M2/Macaulay2/m2/expressions.m2

```diff
@@ -1250,15 +1250,10 @@ net Set := net @@ expression
 texMath Set := texMath @@ expression
 
 -- shortening expressions
-Dots = new Type of Symbol
-cdots=new Dots from symbol cdots
-ddots=new Dots from symbol ddots
-vdots=new Dots from symbol vdots
-ldots=new Dots from symbol ldots
-texMath Dots := x -> "\\" | simpleToString x -- note that \vdots has bad spacing in ordinary LaTeX
-toString Dots := x -> "..."
-net Dots := x -> if x === vdots then "."||"."||"." else if x === ddots then ".  "||" . "||"  ." else "..."
-
+⋯:=symbol ⋯
+⋱:=symbol ⋱
+⋮:=symbol ⋮
+…:=symbol …
 -- used e.g. in chaincomplexes.m2
```

Comment:
I'm confused about what's happening here, and also don't think this particular syntax is good practice.

Reply:
For instance, maybe `cdots = ⋯ = new Operator from symbol ⋯` is preferable, and later in this file it would be better to use the synonym `cdots` rather than `⋯`.

Reply:
well, what was there before was worse, since it defined a sub-Type of `Symbol`, which is not a goo idea in general. this has the merit that it works.

Reply:
but OK with synonyms, will do